.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "challocator"]
+	path = challocator
```

Comment:
A better path would be `allocator/challocator`, in case we wan't to support multiple allocators and let the user choose the one he wish to use via `make config`.

Reply:
weird, I just did the `git submodule add` command. Will see why it ended up like this.